README.md

```diff
@@ -89,6 +89,7 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 0086 | [NFT Metadata Update Oracles](./CIP-0086) | Proposed |
 | 0093 | [Authenticated Web3 HTTP requests](./CIP-0093) | Proposed |
 | 0094 | [SPO On-chain Polls](./CIP-0094) | Active |
+| 0095 | [Web-Wallet Bridge | Governance](./CIP-0095) | Proposed |
```

Comment:
```suggestion
| 0095 | [Web-Wallet Bridge <code>&#124;</code> Governance](./CIP-0095) | Proposed |
```

Reply:
Now I am not feeling so good about the "clean" solution of putting `|` characters in our CIP titles, since they so often need to appear in tables.  This makes it look like we should return to my suggestion (https://github.com/cardano-foundation/CIPs/issues/590#issuecomment-1725919385) of using the keyboard-friendly and typographically neutral minus sign (`-`). 😵‍💫

@Ryun1 @Crypto2099 @Scitz0 what do you think of reverting our consensus today to the good old minus sign... keeping in mind the vast number of unpredictable contexts in which the CIP titles will be repurposed?

Reply:
Agree, let's use `-` to keep it simple.

Reply:
Im happy to revert to `-`